config.hh

```diff
@@ -318,18 +318,17 @@ struct Preferences
   bool enableClipboardHotkey;
   HotKey clipboardHotkey;
 
-  bool enableScanPopup;
   bool startWithScanPopupOn;
   bool enableScanPopupModifiers;
   unsigned long scanPopupModifiers; // Combination of KeyboardState::Modifier
-  bool scanPopupAltMode; // When you press modifier shortly after the selection
-  unsigned scanPopupAltModeSecs;
   bool ignoreOwnClipboardChanges;
 
   bool scanToMainWindow;
   bool ignoreDiacritics;
   bool ignorePunctuation;
 #ifdef HAVE_X11
+  bool trackClipboardScan;
```

Comment:
Does the tracking clipboard still work in Windows?

Reply:
Yes, by default, the scanpopup will always be present. If user click the 💡 on navbar or use the tray icon. 

I think both users and us will have better experience.

Reply:
you mean the `enable scanpopup ` has the same effect as tracking clipboard?